pom.xml

```diff
@@ -833,6 +834,44 @@
         </execution>
       </executions>
     </plugin>
+    <plugin>
+      <groupId>dev.dimlight</groupId>
+      <artifactId>shellcheck-maven-plugin</artifactId>
+      <version>${shellcheck-maven-plugin.version}</version>
+      <configuration>
+        <sourceDirs>
+          <sourceDir>
+            <directory>${project.basedir}</directory>
+            <includes>
+                <include>**/*.sh</include>
+                <include>**/*.bash</include>
+            </includes>
+            <excludes>
+                <exclude>web-client/**</exclude>
+            </excludes>
+          </sourceDir>
+        </sourceDirs>
+        <failBuildIfWarnings>true</failBuildIfWarnings>
+        <binaryResolutionMethod>download</binaryResolutionMethod>
+        <releaseArchiveUrls>
+          <Linux-amd64>
+              https://github.com/koalaman/shellcheck/releases/download/v0.8.0/shellcheck-v0.8.0.linux.x86_64.tar.xz
+          </Linux-amd64>
+          <Mac_OS_X-x86_64>
+              https://github.com/koalaman/shellcheck/releases/download/v0.8.0/shellcheck-v0.8.0.darwin.x86_64.tar.xz
+          </Mac_OS_X-x86_64>
+        </releaseArchiveUrls>
```

Comment:
Is there a reason for using the download method instead of the [embedded](https://github.com/giraciopide/shellcheck-maven-plugin#how-it-works) method? The embedded method might make our downstream builds easier since there's a proxy involved.

Reply:
No real reason other than that it was the default. Will change, thanks for the feedback!